code/__HELPERS/global_lists.dm

```diff
@@ -93,79 +92,52 @@ var/global/list/backbaglist = list("Nothing", "Backpack", "Satchel")
 //////////////////////////
 
 /proc/makeDatumRefLists()
-	var/list/paths
+	// Hair - Initialise all /datum/sprite_accessory/hair into an list indexed by hair-style name
+	init_sprite_accessory_subtypes(/datum/sprite_accessory/hair, hair_styles_list, hair_styles_male_list, hair_styles_female_list)
+
+	// Facial Hair - Initialise all /datum/sprite_accessory/facial_hair into an list indexed by facialhair-style name
+	init_sprite_accessory_subtypes(/datum/sprite_accessory/facial_hair, facial_hair_styles_list, facial_hair_styles_male_list, facial_hair_styles_female_list)
```

Comment:
Could you take out code restructuring changes like this for now. They may be fine later but I'd prefer for large find and replace changes like this we keep the change as pure as possible

Reply:
Except fixing relative pathing, keep those kinds of changes